.github/workflows/ci.yml

```
@@ -206,22 +207,23 @@ jobs:
 
       - name: Define outputs
         id: outputs
-        run: |
-          echo plugin=$(jq -n \
-            --arg id "$(jq -r .id dist/plugin.json)" \
-            --arg version "$(jq -r .info.version dist/plugin.json)" \
-            --arg has-backend "${{ steps.check-for-backend.outputs.has-backend }}" \
-            --arg executable "$(jq -r .executable dist/plugin.json)" \
-            '$ARGS.named'
-          ) >> "$GITHUB_OUTPUT"
-
-          echo universal-zip='${{ steps.universal-zip.outputs.zip }}' >> "$GITHUB_OUTPUT"
-          echo os-arch-zips='${{ steps.os-arch-zips.outputs.zip }}' >> "$GITHUB_OUTPUT"
-
-          # combine universal zip string with os-arch zips into a single array
-          zips=$(echo -e '["${{ steps.universal-zip.outputs.zip }}"]\n${{ steps.os-arch-zips.outputs.zip }}' | jq -cs 'add')
-          echo zips="$zips" >> "$GITHUB_OUTPUT"
-
+        run: |          
+          {
+            echo plugin="$(jq -n \
+                --arg id "$(jq -r .id dist/plugin.json)" \
+                --arg version "$(jq -r .info.version dist/plugin.json)" \
+                --arg has-backend "${{ steps.check-for-backend.outputs.has-backend }}" \
+                --arg executable "$(jq -r .executable dist/plugin.json)" \
+                '$ARGS.named'
+              )"
```

I think this may break the workflow because the JSON should be in one line.

Previous output:
```
plugin={ "id": "example-httpbackend-datasource", "version": "1.0.0", "has-backend": "false", "executable": "gpx_datasource_http_backend" }
```

New output:

```
plugin={
  "id": "example-httpbackend-datasource",
  "version": "1.0.0",
  "has-backend": "false",
  "executable": "gpx_datasource_http_backend"
}
```

This can easily be fixed by adding `-c` to `jq`:

```suggestion
            echo plugin="$(jq -n -c \
                --arg id "$(jq -r .id dist/plugin.json)" \
                --arg version "$(jq -r .info.version dist/plugin.json)" \
                --arg has-backend "${{ steps.check-for-backend.outputs.has-backend }}" \
                --arg executable "$(jq -r .executable dist/plugin.json)" \
                '$ARGS.named'
              )"
```

Which produces the following output:

```
plugin={"id":"example-httpbackend-datasource","version":"1.0.0","has-backend":"false","executable":"gpx_datasource_http_backend"}
```